lambda/custom/pocket_strings.json

```diff
@@ -4,8 +4,8 @@
       "LINK_ACCOUNT": "To start using this skill, please use the companion app to authenticate on Amazon. ",
       "WELCOME_MSG": "Welcome to Pocket. To begin, say get articles.",
       "WELCOME_REPROMPT": "Begin by saying get my articles.",
-      "START_HELP": "To listen to an article, say “Play”, followed by the title or a few keywords. For example, say “Play the one about polar bears.” You can say “Get my articles” to hear the first few articles in your Pocket queue.",
-      "TITLE_HELP": "To listen to an article, say “Play”, followed by the title or a few keywords. For example, say “Play the one about polar bears.” You can say “Get my articles” to hear the first few articles in your Pocket queue, next, to hear the next few, and repeat, to hear them again. ",
+      "START_HELP": "To listen to an article, say Play, followed by the title or a few keywords. For example, say Play the one about polar bears. You can say Get my articles to hear the first few articles in your Pocket queue.",
+      "TITLE_HELP": "To listen to an article, say Play, followed by the title or a few keywords. For example, say Play the one about polar bears. You can say Get my articles to hear the first few articles in your Pocket queue, next, to hear the next few, and repeat, to hear them again.",
```

Comment:
The tests on Help strings were failing -- my best guess is that it was related to the quotation characters embedded in the strings in the previous version of the file. So I rewrote the strings omitting the funky quote marks and it was happy again. But I'm not sure why they started failing only after this re-organization.